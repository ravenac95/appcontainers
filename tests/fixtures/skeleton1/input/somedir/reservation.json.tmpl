{
    "name": "{{reservation.name}}",
    "ip": "{{reservation.ip}}",
    "mac": "{{reservation.mac}}"
}
