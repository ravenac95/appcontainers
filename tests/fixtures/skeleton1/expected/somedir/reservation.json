{
    "name": "SOMENAME",
    "ip": "192.168.0.1",
    "mac": "00:16:3e:00:00:01"
}
